.github/workflows/kind-e2e.yaml

```diff
@@ -49,6 +49,17 @@ jobs:
         - registry-name: registry.local
           registry-port: 5000
           registry-url: registry.local:5000
+        # Run the v1alpha1 conformance suite with alpha tests enabled.
+        # This is explicitly added since we don't want to run every single matrix element twice.
+        - k8s-version: v1.19.1
+          test-suite: ./test/conformance/api/v1alpha1
+          test-flags: "--enable-alpha"
```

Comment:
I'd expect us to just add this to existing configurations.  I don't think that duplicating works properly based on prior experimentation 🤔

Reply:
looks like it does in https://github.com/julz/serving/pull/2/checks?check_run_id=1260072669

Reply:
(it works because `test-suite: ./test/conformance/api/v1alpha1` isn't in the existing matrix, so it's a new configuration)

Reply:
Yeah, I was actually just coming back to leave that comment 🤣 

I think what we really want is to add `./test/conformance/api/v1alpha1` to the `test-suite` list and then have a separate block with:
```
- test-suite: ./test/conformance/api/v1alpha1
  test-flags: --enable-alpha
```

This should give us a new leg for all K8s versions, which has the test-flags.  If there are combinations that don't work, I believe we can `exclude:` them (likely with `TODO(#1234)` 🙏 ), but my preference would be to add this for as many as we can here.

Reply:
that's fine with me, I was guessing we'd want to limit the test explosion and only add for one combination (like we do with ingress), will update

Reply:
When I added these we had alpha and beta, so it's sort of a return to that, but with many fewer tests (this is doing to be so fast 😂 )

Reply:
rebased & updated, PTAL :)